src/storage/src/source/generator.rs

```diff
@@ -139,7 +147,7 @@ impl SourceRender for LoadGeneratorSourceConnection {
                         let message = (
                             output,
                             Ok(SourceMessage {
-                                key: Row::default(),
+                                key: Row::pack_slice(&[Datum::UInt64(1)]),
```

Comment:
Why did we change the schema of the other generators?

Reply:
oh this was a testing thing. I think in general we drop this key but it may be the flake? not sure